src/Playroom/CodeEditor/keymaps/comment.ts

```diff
@@ -198,6 +198,19 @@ function getUpdatedContent(existingContent: string, range: TagRange) {
 
 type CommentType = 'line' | 'block';
 
+const isOnlyWhitespace = (input: string) => /^\s+$/.test(input);
+
+const isFullExpressionSlot = (tokens: CodeMirror.Token[]) => {
+  const formattedLineTokens = tokens.filter(
+    (token) => token.type !== 'comment' && !isOnlyWhitespace(token.string)
+  );
+
+  return (
+    formattedLineTokens.at(0)?.string === '{' &&
+    formattedLineTokens.at(-1)?.string === '}'
```

Comment:
Am I right that this is only intended to work on one line?

For example:
```ts
{() => {}
}
<div>First line</div>
<div>Second line</div>
<div>Third line</div>
```

Commenting out line 1 of this will result in (renders the closing curly in the frame):
```ts
{/* {() => {} */}
}
<div>First line</div>
<div>Second line</div>
<div>Third line</div>
```

Thoughts?

Reply:
I don't think there's any way to comment out line 1 that doesn't result in a syntax error, and I don't think the plugin should go beyond your selection to try and make your comment non-breaking. You just messed up the comment, at that point

Reply:
Interesting, yeah it's only meant to work on the lines selected so in this case it's working as intended. I agree with Ben's point although it's annoying to get the closing curly rendered 😅